app/Helpers/database/forum.php

```diff
@@ -305,7 +305,7 @@ function generateGameForumTopic(User $user, int $gameId): ?ForumTopicComment
     $wikipediaURL = "https://www.google.com/search?q=site:en.wikipedia.org+$urlSafeGameTitle";
 
     $topicPayload = "Official Topic Post for discussion about [game=$gameId]\n" .
-        "Created " . date("j M, Y H:i") . " by [user={$user->User}]\n\n" .
+        "Created " . date("j M, Y H:i") . " by [user={$user->username}]\n\n" .
```

Comment:
This should probably be `$user->display_name`.

Reply:
Fixed in latest.